### ISSUE TITLE
Decorators Adjustment

### ISSUE THREAD

**Issue by jleeson** [OutwalkStudios/exalt#10]
Currently we have the `define` and `options` decorators. these could ideally be merged to support using options inside the define decorator.

**Current Syntax**
```js
@define("my-clock")
@options({ shadow: true })
```

**Proposed Syntax**
```js
@define("my-clock")
```
or

```js
@define({ tag: "my-clock", shadow: true })
```

**Comment by jleeson** [comment]
This syntax change will be released in v0.5.1 for bundle size and performance improvements. this also allows for reduced boilerplate so its an all around improvement.

**Comment by jleeson** [comment]
this has now been released with @exalt/core v0.5.1
